resources/views/partials/forms/edit/image-upload.blade.php

```diff
@@ -1,9 +1,9 @@
-<div class="form-group {{ $errors->has('image') ? 'has-error' : '' }}">
+<div class="form-group {{ $errors->has((isset($fieldname) ? $fieldname : 'image')) ? 'has-error' : '' }}">
```

Comment:
@inietov this just accepts the variable that we're passing to the blade partial now